test/e2e/build_test.go

```diff
@@ -668,4 +668,23 @@ RUN ls /dev/test1`, ALPINE)
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
 	})
+
+	It("podman build remote image test", func() {
+		if IsRemote() {
+			podmanTest.StopRemoteService()
+			podmanTest.StartRemoteService()
+		} else {
+			Skip("Only valid at remote test")
+		}
+
+		targetPath, err := CreateTempDirInTempDir()
+		Expect(err).To(BeNil())
+		containerFile := filepath.Join(targetPath, "Containerfile")
+		content := `FROM scratch`
+		Expect(ioutil.WriteFile(containerFile, []byte(content), 0755)).To(BeNil())
+
+		session := podmanTest.Podman([]string{"build", "-t", "test", targetPath})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
```

Comment:
Should check the output to make sure you get AUX correct?

Reply:
@rhatdan I think its the API output , `podman` client ignores that so would not be possible in `e2e` tests. I think test for `AUX` is only possible here  https://github.com/containers/podman/blob/main/test/apiv2/10-images.at

Reply:
@ananthb If you try squeezing a test for `aux` https://github.com/containers/podman/blob/main/test/apiv2/10-images.at then it would be great.

Reply:
Done